File: Contraseña.cpp
#include <iostream>
#include <string.h> // Esta libreria permite usar "strcmp"

using namespace std;

int main (){
    int i = 1;
    char contrareal[]="Hola999", contra [20], usuario[20];
    
    cout << "¡Bienvenido a Facebook!" << endl << endl << "Usuario: ";
    cin >> usuario;
    proceso:
    cout << "Contraseña: ";
    cin >> contra;

    if (strcmp(contrareal,contra) == 0){ // "strcmp" sirve para comparar cadenas (segun lo que entendí), si se igualan a 1 son distintas...
        cout << endl;
        cout << "Contraseña correcta";
    } 
    else{
        while (i < 4){
            cout << "Error de credenciales" << endl << endl;
            i += 1;
            goto proceso;
        }

        if (i == 4){
            cout << "Su conexion no es la adecuada" << endl << endl;
            i += 1;
            goto proceso;
        }

        if (i == 5){
            cout << "Su cuenta no existe";
        } 
    }

    return 0;
}

File: Contraseña.py
print ("¡Bienvenido!")
usuario = input ("Ingrese su usuario: ")
contra = 0
intentos = 1

while (contra != "Hola999" and intentos <= 5):
    contra = input ("Ingrese la contraseña: ")

    if contra == "Hola999":
        intentos = 6
        print (" ")
        print ("¡Contraseña correcta!")

    if intentos == 1 or intentos == 2 or intentos == 3:
        print (" ")
        print ("Error de credendiales")
        print (" ")
    if intentos == 4:
        print (" ")
        print ("Su conexion no es la adecuada")
        print (" ")
    if intentos == 5:
        print (" ")
        print ("Su cuenta no existe")
        print (" ")
    intentos = intentos + 1